### PR TITLE
Add e2e_simple for v2 and enable in circle

### PR DESCRIPTION
**Pull request by ostromart**
Add support for gateway, port over rules.

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -99,6 +99,42 @@ jobs:
       - store_test_results:
           path: /go/out/tests
 
+  e2e-simple-v1alpha3-v2:
```
Comment (costinm) [inline .circleci/config.yml]: How about:
- call it 'e2e-simple'
- remove the old e2e-simple, which is using v1

We need to remove the old one either way.

**Comment by ostromart** [comment]
Ok, I removed the old e2e_simple. How about I create a new PR to remove all the v1 tests? That is, make the old targets point to the v2 tests.

**Comment by costinm** [comment]
+1 on removing all remaining v1 tests ( and then code ). We still use few files in v1 for agent, will also be gone when SDS is ready ( I suspect we can already remove watching the secretes and restarting envoy - my understanding is that envoy can already watch and reload the secrets, someone in security should tested if it is true, would allow us to reduce refresh interval)

**Comment by costinm** [comment]
But separate PRs...

**Pull request merged**